tests/unit/folio/flow/test_flow.py

```
@@ -0,0 +1,65 @@
+    return done(Survey(choice=choice, approved=approved, note=note))
+
+
+@ritual("survey")
+async def survey() -> Transition:
+    await asyncio.sleep(0)
+    return goto(gather, State())
+
+
+class TestDecideMedium:
+    @staticmethod
+    def test_choice_confirm_free_round_trip_via_stdin():
```

Ruff (0.15.21)

[warning] Missing return type annotation for staticmethod `test_choice_confirm_free_round_trip_via_stdin`

Add return type annotation: `None`

(ANN205)